conda/meta.yaml

```diff
@@ -23,10 +19,9 @@ test:
     - python
     - bin
     - tests
-    - setup.cfg
     - pyproject.toml
   commands:
-    - py.test
+    - pytest
 
 requirements:
   host:
```

Comment:
Can you please update lines 33 and 39 bellow with:

```
- python {{ python }}
```
?

This will make sure it will catch the python version we are using for the build.

Also, you can remote `pytest-runner` from the "host" dependencies.